assignment2.txt

```diff
@@ -0,0 +1,13 @@
+Clarke Littlejohn
+
+1) Yes there can be two or more operators in a single line of code.
+
+2) logical erros cause operations to execute in the wrong order. Syntax erros is when the code is can not be used at all the code in
```

Comment:
Correct, it's also important to note that syntax errors occur during compile time whereas logical errors occur during runtime.